static/js/components/tiles/disaster_event_map_tile.tsx

```diff
@@ -67,6 +67,12 @@ const MAP_POINTS_MIN_RADIUS_EARTH = 0.8;
 // Set of dcids of places that should use the selected place as the base geojson
 const PLACE_MAP_PLACES = new Set(["country/AUS", "country/BRA"]);
 
+// Stores IDs for the zoom buttons on the map
+interface ZoomParams {
```

Comment:
don't need to re-define this: https://github.com/datacommonsorg/website/blob/master/static/js/chart/draw_d3_map.ts#L46